mlibrary/lib.umich.edu
# [WEBSITE-87] Rebuild Staff Directory Table in CSS

# Overview
This pull request resolves [WEBSITE-87](https://mlit.atlassian.net/browse/WEBSITE-87).

The original [Staff Directory](https://www.lib.umich.edu/about-us/staff-directory) table was split into two components: one for desktop and one for small screens. Rendering the component depended on JavaScript to determine the size of the browser's window. The desktop component has been rewritten to become responsive natively using just CSS. It is now far more fluid in sizing and requires less loading.

## What's different?

### Table Header
The table header now has a thicker bottom border, to be consistent with the Hours and Locations table, along with other tables relating to the Design System styling.

### `WindowResize()`
This function has been removed since it is no longer used.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Compare [Staff Directory](http://localhost:8000/about-us/staff-directory) to the live site. Are they styled the same? Do they function the same?
